tox.ini

```diff
@@ -80,11 +80,11 @@ deps =
     oldestdeps: asdf==2.8.*  # asdf-astropy==0.3.*
     oldestdeps: asdf-astropy==0.3.*
     oldestdeps: attrs < 24.1.0  # ASDF_LT_3_4
-    oldestdeps: scipy==1.8.*
-    oldestdeps: pyyaml==3.13
+    oldestdeps: scipy==1.9.*
+    oldestdeps: pyyaml==6.0.0
     oldestdeps: ipython==7.32
     oldestdeps: pandas==2.0.*
-    oldestdeps: pyarrow==7.0.0
+    oldestdeps: pyarrow==10.0.1
```

Comment:
`pyarrow>=7.0.0` in `pyproject.toml` needs to be updated to be consistent with this change here.

Reply:
done